src/main/java/utility/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package utility;
+
+import java.util.Random;
+import model.Color;
+import model.Figure;
+import model.figure.Circle;
+import model.figure.IsoscelesTrapezoid;
+import model.figure.Rectangle;
+import model.figure.RightTriangle;
+import model.figure.Square;
```

Comment:
The import statements should be grouped together without empty lines between them.